.github/workflows/stale.yaml

```diff
@@ -0,0 +1,20 @@
+name: 'Label stale PR'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 7 days with no activity.'
+          days-before-pr-stale: 7
+          days-before-pr-close: -1
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity.'
+          days-before-issue-stale: 30
+          days-before-issue-close: -1
```

Comment:
```suggestion
          days-before-issue-close: 1
```

Reply:
If we do this, we should probably update the message to something like 
`This issue is stale because it has been open 30 days with no activity. This project switched to Jira at https://issues.redhat.com/projects/RHIDP/ for issue tracking. If this issues is still relevant, please reopen it there.'

Reply:
I changed it 7 days be a little bit nicer